wrappingGsdk/wrapper/Program.cs

```diff
@@ -38,8 +44,40 @@ static void Main(string[] args)
         // starts main game process and wait for it to complete
         public static void InitiateAndWaitForGameProcess(string gameserverExe, IEnumerable<string> args)
         {
-             // here we're starting the script that initiates the game process
-            gameProcess = StartProcess(gameserverExe, args);
+            activeConfig = GameserverSDK.getConfigSettings();
+
+            // here we're starting the script that initiates the game process
+            if (activeConfig.TryGetValue(portName, out string listeningPortString))
+            {
+                GameserverSDK.LogMessage($"{portName}:{listeningPortString} was found in GSDK Confg Settings.");
+                _listeningPort = listeningPortString;
+            }
+            else
+            {
+                LogMessage($"Cannot find {portName} in GSDK Config Settings. Please make sure the MockAgent is running " +
+                             $"and that the MultiplayerSettings.json file includes correct {portName} as a GamePort Name.");
```

Comment:
Is it GamePort or RealPort?

Reply:
We are using `game_port` as a port name for MPS samples, and using `RealPort` for our functional tests and GSDK samples (e.g WinRunner).
For the consistency of MPSSample, I will use game_port as port name if that makes sense.